Britannia/localization/english/BTWK_buildings_l_english.yml

```diff
@@ -141,3 +157,31 @@
  building_type_antonine_wall_01_desc:0 "The Antonine Wall was constructed around a quarter of a century after Hadrian's Wall, during the reign of the Emperor Antoninus Pius. While it was intended to be a new, more northernly Caledonian frontier, it was occupied for only a few years before its desertion. "
  building_antonine_wall_01:0 "$building_type_antonine_wall_01$"
  building_antonine_wall_01_desc:0 "$building_type_antonine_wall_01_desc$"
+
+ ################
+ # BATH'S BATHS #
+ ################
+ building_type_btwk_aquae_sulis_baths_01:0 "Roman Baths"
```

Comment:
Possibly "Baths of Aquae Sulis"? Roman Baths were in every Roman city tbh

Reply:
Was going to put that, but again thought we were localising using English names so avoided using it (as Baths of 'Bath' sounds well stupid lol)

Reply:
Hmm I guess my thought was that Aquae Sulis is the standard English name for Bath in the Roman era

Reply:
That's fine. I've changed it to Aquae Sulis anyway as it does sound correct. I just misunderstood what you meant by using 'English' names, which has been resolved by your comment on Alt Clut/Strathclyde

Reply:
I guess by "English name" I really mean "English historiographical name", i.e. the name you're mostly likely to see when opening a paper or textbook or encyclopedia.